ecocode-rules-specifications/src/main/java/io/ecocode/rules/java/JavaRulesRepository.java

```diff
@@ -0,0 +1,8 @@
+package io.ecocode.rules.java;
+
+public class JavaRulesRepository {
+  public static final String REPOSITORY_KEY = "ecocode-java";
+  public static final String NAME = "ecoCode";
+  public static final String LANGUAGE = "java";
+  public static final String RESOURCE_BASE_PATH = JavaRulesRepository.class.getPackageName().replace('.', '/') + "/specifications";
```

Comment:
Hi, are your sure about this path ? When I run the test file, the ressource path is _io/ecocode/rules/java/specifications_ , there is noting there 🤔
And I don't know where it's define but it still search html files (_io/ecocode/rules/java/specifications/EC27.html_)

Reply:
This path is "generated" with Maven build based on resources in `ecocode-rules-specifications/src/main/rules`.